src/codeflare_sdk/cluster/auth.py

```diff
@@ -162,7 +162,14 @@ def config_check() -> str:
     global config_path
     global api_client
     if config_path == None and api_client == None:
-        config.load_kube_config()
+        try:
+            config.load_kube_config()
+        except config.config_exception.ConfigException:
+            try:
+                config.load_incluster_config()
+            except:
```

Comment:
Anyway to avoid this nested `try/except` statement?  Can we check that we are in the cluster before trying to load the config?

Reply:
Added a check to see if the config file exists and if not a check to see if we are in the cluster before using `load_incluster_config`

Reply:
Thanks I think that's much better. I'll test it this morning.